client/pyscripts/fm11rf08_full.py

```diff
@@ -0,0 +1,942 @@
+                        ab cd ef
+'''
+'''
+ ,-------------------.
+(  2.2 : ACCESS BITS  )
+ `-------------------'
+
+	The Access bits on both (used) Sectors is the same:  78 77 88
+
+	Let's reorganise that according to the official spec Fig 9.
+	     Access        C1 C2 C3
+	   ==========     ===========
+	    78 77 88  -->  78 87 87
+	    ab cd ef  -->  cb fa ed
+
+	The second nybble of each byte is the inverse of the first nybble.
+	It is there to trap tranmission errors, so we can just ignore it/them.
+
+	So our Access Control value is : {c, f, e} == {7, 8, 8}
+
+	Let's convert those nybbles to binary
+		(c) 7 --> 0111
+		(f) 8 --> 1000
+		(e) 8 --> 1000
+		          |||| ...and transpose them:
+		          ||||
+		          |||`--- 100 - Block 0 Access bits
+		          ||`---- 100 - Block 1 Access bits
+		          |`----- 100 - Block 2 Access bits
+		          `------ 011 - Block 3 Access bits [Sector Trailer]
+
+	Now we can use the lookup table [Table 3] to work out what we can do
+	with the Sector Trailer (Block(S,3)):
+	
+		      |    Key A     | | Access Bits  | |    Key B     |
+		      | read ¦ write | | read ¦ write | | read ¦ write |
+		      +------¦-------+ +------¦-------+ +------¦-------+
+		000 : |  --  ¦ KeyA  | | KeyA ¦  --   | | KeyA ¦ KeyA  |
+		001 : |  --  ¦ KeyA  | | KeyA ¦ KeyA  | | KeyA ¦ KeyA  | Transport Mode
+		010 : |  --  ¦  --   | | KeyA ¦  --   | | KeyA ¦  --   |
+
+		011 : |  --  ¦ KeyB  | | A+B  ¦ KeyB  | |  --  ¦ KeyB  | <-- Our Card!
+
+		100 : |  --  ¦ KeyB  | | A+B  ¦ --    | |  --  ¦ KeyB  |
+		101 : |  --  ¦  --   | | A+B  ¦ KeyB  | |  --  ¦  --   |
+		110 : |  --  ¦  --   | | A+B  ¦  --   | |  --  ¦  --   | }__ 
+		111 : |  --  ¦  --   | | A+B  ¦  --   | |  --  ¦  --   | }   The Same!?
+		
+	Our card uses 011, for (both of) the (used) Sector Trailer(s). So: 
+		Both Key A and Key B can READ the Access Bits
+		Key B can (additionally) WRITE to Key A, Key B (itself), and the Access Bits
+
+	Then we can do a similar lookup for the 3 data Blocks (in this Sector)
+	This time using [Table 4]
+
+		      |    Data      |   Counter   |
+		      | read ¦ write | Inc  ¦ Dec  |
+		      +------¦-------+------¦------+
+		000 : | A+B  ¦  A+B  | A+B  ¦  A+B | Transport Mode
+		001 : | A+B  ¦  --   |  --  ¦  A+B |
+		010 : | A+B  ¦  --   |  --  ¦  --  |
+		011 : | KeyB ¦  KeyB |  --  ¦  --  |
+
+		100 : | A+B  ¦  KeyB |  --  ¦  --  | <-- Our Card!
+
+		101 : | KeyB ¦  --   |  --  ¦  --  |
+		110 : | A+B  ¦  KeyB | KeyB ¦  A+B |
+		111 : | --   ¦  --   |  --  ¦  --  |
+
+	Our card uses 100, for all of the (used) Sectors. So: 
+		Both Key A and Key B can READ the Block
+		Only Key B can WRITE to the Block
+		The block cannot be used as a "counter" because:
+			Neither key can perform increment nor decrement commands
+
+	WARNING: 
+		IF YOU PLAN TO CHANGE ACCESS BITS, RTFM, THERE IS MUCH TO CONSIDER !
+'''
+#==============================================================================
+def dumpAcl():
```

Comment:
it would be interesting to isolate this function in a separate file to be imported, so other ppl can use it in their scripts s well, or call the script with a provided block0.
Note that we already have `hf mf acl`

Reply:
qv. https://github.com/RfidResearchGroup/proxmark3/pull/2613#discussion_r1830270895

this function requires data other than block 0

I was originally going to call `hf mf acl` (as I say, this script is initially to support a defcon workshop) and parse the output (if I had to), but in the end I found the output difficult to decipher, and tricky to parse - in the end, some bit twiddling just seemed easier.